src/obis-observer/utils/obis.ts

```diff
@@ -37,3 +52,46 @@ export const toString = ( obis: IObis ): string => {
 
     return result;
 };
+
+
+/**
+ * Convert string to object.
+ *
+ * @param obisString string according to the mask A-B:C.D.E*F
+ */
+export const fromString = ( obisString: string ): IObis => {
+    const result: THashTable = {};
+    let unprocessed = obisString;
+    let parts;
+
+    parts = unprocessed.split('-');
+    if ( parts.length > 1 ) {
```

Comment:
```suggestion

    if ( parts.length > 1 ) {
```

?

Reply:
I prefer this way as it's a single block